### DISCUSSION TITLE
expose a ride's "covered" status in UI

### DISCUSSION THREAD

**Discussion by cheweytoo** [OpenRCT2/OpenRCT2#21955] [Feature requests]
Inspired by https://github.com/OpenRCT2/OpenRCT2/discussions/21938 – a thing I've dreamt of having for a long time: A ride's properties showing whether that ride is considered "covered" by the game. Maybe even the actual percentage, and how far it is from the threshold.

At the moment, it's complete guesswork when building, and can only be deduced from guest behaviour after opening the ride.

**Comment by autosysops** [comment]
Agreed, this data is already available in the object. it's just a matter of reading if the flag is set and showing it here
![image](https://github.com/OpenRCT2/OpenRCT2/assets/96066335/d611f3ff-4f31-4263-9870-84a393bfdd46)

I think the value is set on "car" level and not ride level so it would need to be shown here. Think this would expose some nice information to the player which is hidden, and that's always good.
